src/UI/class-admin-columns-parsely-stats.php

```diff
@@ -223,6 +240,20 @@ public function get_parsely_stats_response( $analytics_api ) {
 			return null;
 		}
 
+		if ( ! $this->parsely->api_secret_is_set() ) {
+			return array(
+				'data'  => null,
+				'error' => array(
```

Comment:
Can/should we make the messages translatable?

Reply:
Oops ... Done